src/locales/en.json

```diff
@@ -1273,7 +1274,7 @@
       "endDate": "Estimated end date for work",
       "workArea": "Work area",
       "workAreaPlural": "Work areas",
-      "workAreaOverlapsSingle": "The work area extends beyond the boundary of the cable report; submit a <a>{{{applicationIdentifier}}</a> change notification to the cable report.",
+      "workAreaOverlapsSingle": "The work area extends beyond the boundary of the cable report <a>{{{applicationIdentifier}}</a>. Change the boundary of the work area or apply for a new cable report.",
```

Comment:
There seems to be an extra `{` here:

![Screenshot 2025-04-28 at 9 27 40](https://github.com/user-attachments/assets/164e5474-7f1c-4e05-95e6-10cd4d03a893)